docs/application/dotnet/guides/nui/tizenshell.md

```diff
@@ -0,0 +1,43 @@
+# Tizen Window System Shell
+Tizen Window System Shell (TZSH) is an interface for manipulating windows of the system Graphical User Interface (GUI) services such as Quickpanel. It provides C and C# libraries that allow you to control windows of system GUI services and get notifications about the state changes of each window within an application.
+
+The system GUI service runs different processes. Therefore, all the operations of TZSH are performed using the inter-process communication (IPC). It is needed to enable communication between the system GUI service processes and separated user application process using the display server. The following figure illustrates a simple software architecture of TZSH:
+
+![Layer diagram for the Tizen window system shell](./media/tzsh_arch.png)
+
+To support each system GUI service in Tizen, TZSH provides the following libraries:
+* **[Quickpanel](./quickpanelclient.md)**: For the Quickpanel window that shows notifications and system setup widgets.
+Most applications do not need to use the TZSH’s functionalities. However, in some cases, certain applications may require to perform manipulation of system GUI service window. For example, media player application needs to close the Quickpanel window during playback of video. In this case, you can use the Quickpanel client library.
+
+## Prerequisites
+To use the functions and structures of the TZSH API, using `Tizen.NUI.WindowSystem` in your application:
```

Comment:
To use the functions and structures of the TZSH API, use`Tizen.NUI.WindowSystem` in your application:

Reply:
Done.